keps/sig-node/2129-remove-cadvisor-json-metrics/README.md

```diff
@@ -174,6 +174,7 @@ Remove the `--enable-cadvisor-json-endpoints` flag and the kubelet stops serving
 
 - 2020-10-04: Initial version of the KEP
 - 2021-02-04: Updates based on feedback
+- 2021-03-10: Implementation code complete
```

Comment:
would be great to link to the relevant PRs :)

Reply:
https://github.com/kubernetes/kubernetes/pull/99236
Fine.